pkg/invoice/invoice_test.go

```diff
@@ -50,14 +50,14 @@ func (s *InvoiceSuite) SetupSuite() {
 
 	require.NoError(s.T(),
 		db.GetNamed(tdb, &s.memoryProduct,
-			"INSERT INTO products (source,target,amount,unit,during) VALUES (:source,:target,:amount,:unit,:during) RETURNING *", db.Product{
+			"INSERT INTO products (source,target,type,amount,unit,during) VALUES (:source,:target,:type,:amount,:unit,:during) RETURNING *", db.Product{
 				Source: "test_memory:us-rac-2",
 				Amount: 3,
 				During: db.InfiniteRange(),
 			}))
 	require.NoError(s.T(),
 		db.GetNamed(tdb, &s.storageProduct,
-			"INSERT INTO products (source,target,amount,unit,during) VALUES (:source,:target,:amount,:unit,:during) RETURNING *", db.Product{
+			"INSERT INTO products (source,target,type.amount,unit,during) VALUES (:source,:target,:type,:amount,:unit,:during) RETURNING *", db.Product{
```

Comment:
```suggestion
			"INSERT INTO products (source,target,type,amount,unit,during) VALUES (:source,:target,:type,:amount,:unit,:during) RETURNING *", db.Product{
```

Tests are failing because of this

Reply:
Thanks. I just didn't have time after I asked for a review to fix the tests!